### ISSUE TITLE
Can't send messages

### ISSUE THREAD

**Issue by DevOFVictory** [Auties00/WhatsappWeb4j#35]
```java
api.sendMessage(chat, new TextMessage("Hello World"), info);
```
seems not to work. Everything else (login, listeners etc.) worked as expected, but this method does nothing, without printing an error. I also tried the examples in the repo, but they worked neither.

**Comment by Auties00** [comment]
> ```java
> api.sendMessage(chat, new TextMessage("Hello World"), info);
> ```
> 
> seems not to work. Everything else (login, listeners etc.) worked as expected, but this method does nothing, without printing an error. I also tried the examples in the repo, but they worked neither.

Can you show me the full snippet?

**Comment by DevOFVictory** [comment]
Yeah, indeed
```java
    @Override
    public void onNewMessage(@NonNull Chat chat, @NonNull MessageInfo info) {

        if (info.container().textMessage().text().equalsIgnoreCase("*test")) {
            System.out.println("debug");
            api.sendMessage(chat, new TextMessage("Hello World!"), info);
            api.sendMessage(chat, "Hello World!", info);
        }

    }
```

**Comment by nekiak** [comment]
same thing here, can't send messages

**Comment by Auties00** [comment]
> ```java
> api.sendMessage(chat, new TextMessage("Hello World"), info);
> ```
> 
> seems not to work. Everything else (login, listeners etc.) worked as expected, but this method does nothing, without printing an error. I also tried the examples in the repo, but they worked neither.

I can confirm that the only unit tests that fail regard messages, once I setup github's actions correctly such issues should be caught much sooner

**Comment by Auties00** [comment]
@DevOFVictory 
@nekiak 
The issue should be fixed, check the new release(2.1.3).

**Comment by DevOFVictory** [comment]
Nice, thanks!
